src/modules/notifications/notifications.resolver.ts

```diff
@@ -0,0 +1,13 @@
+import { Context, Query, Resolver } from '@nestjs/graphql';
+import { NotificationsService } from './notifications.service';
+import { Notification } from './entities/notification.entity';
+
+@Resolver()
+export class NotificationsResolver {
+    constructor(private readonly notificationsService: NotificationsService) {}
+
+  @Query(() => [Notification], { name: 'notifications' })
+  async findAll(@Context() context) {
```

Comment:
Add proper example in description I see **# Write your query or mutation here**

Reply:
Fix lint error

Reply:
Fixed in the commit https://github.com/OsmosysSoftware/osmo-notify/pull/68/commits/07cbc5def15a46d0b95364563c2c2c035efbfe18

Reply:
> Add proper example in description I see **# Write your query or mutation here**

My bad, that was a comment that got exported in curl, Removed it.
Added a screenshot in description as well.